CIP-1694/README.md

```diff
@@ -171,8 +171,8 @@ Note that a motion of no-confidence is an extreme measure that enables Ada holde
 Governance actions are **ratified** through on-chain voting actions.  Different kinds of governance actions have different ratification requirements: depending on the type of governance action, an action will become ratified if some specific combination of the following occurs:
 
 * the Constitutional Committee approves of the action (`Quorum-Many` members vote 'yes');
-* the DReps approve of the action (the stake controlled by the DReps who vote 'yes' meets a certain threshold over those who vote 'no');
-* the SPOs approve of the action (the stake controlled by the SPOs who vote 'yes' meets a certain threshold over those who vote 'no').
+* the DReps approve of the action (the stake controlled by the DReps who vote 'yes' meets a certain threshold over the total registered voting stake);
```

Comment:
I think the confusing wording here is your use of "over". I think perhaps you want to say "a threshold _of_ the total registered voting stake".